packages/@ember/helper/index.ts

```diff
@@ -314,3 +314,4 @@
 
 export { setHelperManager, helperCapabilities as capabilities } from '@glimmer/manager';
 export { invokeHelper, hash, array, concat, get, fn } from '@glimmer/runtime';
+export { uniqueId } from '@ember/-internals/glimmer';
```

Comment:
It seems like module resolution doesn't let me re-export this from `@ember/-internals/glimmer/lib/helpers.../` directly. So first I had to export it from the `-internals/glimmer` index first.